web_src/js/components/RepoContributors.vue

```diff
@@ -88,6 +91,9 @@ export default {
   methods: {
     sortContributors() {
       const contributors = this.filterContributorWeeksByDateRange();
+      const min = dayjs(this.xAxisMin).format('YYYY-MM-DD');
+      const max = dayjs(this.xAxisMax).format('YYYY-MM-DD');
+      this.searchQuery = `${this.repoLink}/commits/branch/${this.repoBranch}/search?q=after:${min}, before:${max}, author:`;
```

Comment:
Do we need to `encodeURIComponent(this.repoBranch)`? Try with a branch name that contains a space character.

Reply:
> Actually this.repoBranch might need to use path segment escape.

There is no path escape function in JS, I think `encodeURIComponent` is suitable for alle cases, path or query.

Reply:
I guess "org/repo/branch/feature%2Fbranch" doesn't work.

Reply:
Yes, you can not pass `/` to `encodeURIComponent`, it's mean for individual segments or search param values.